src/probeinterface/probe.py

```diff
@@ -932,7 +943,7 @@ def to_image(self, values, pixel_size=0.5, num_pixel=None, method="linear", xlim
 
         return image, xlims, ylims
 
-    def get_slice(self, selection):
+    def get_slice(self, selection: np.ndarray[Union[bool, int]]):
```

Comment:
What do you think of bool | int notation. I think is cleaner. It should work in our minimal supported python with from __future__ import annotations.

Reply:
https://peps.python.org/pep-0604/

Reply:
Ok.